roles/kubernetes/node/defaults/main.yml

```diff
@@ -28,17 +28,25 @@ kubelet_fail_swap_on: true
 # Reserve this space for kube resources
 kube_memory_reserved: 256Mi
 kube_cpu_reserved: 100m
+## A dictionary of extra config variables to add to kubeReserved, e.g.
+##  kube_reserved_extra_vars: {
```

Comment:
An example should be added to the sample inventory and some reference to this capability in the docs.

Reply:
Which file should I add the document to?

Reply:
I've added some docs here https://github.com/kubernetes/kubernetes/pull/110323.